packages/babel-preset-latest/README.md

```diff
@@ -36,9 +36,11 @@ require("babel-core").transform("code", {
 
 ### Options
 
-- `es2015`: Optionally not run any plugins from this preset (defaults to true)
-- `es2016`: Optionally not run any plugins from this preset (defaults to true)
-- `es2017`: Optionally not run any plugins from this preset (defaults to true)
+### `es2015`
+
+`boolean`, defaults to `false`.
+
+Toggles including plugins from the [es2015 preset](/docs/plugins/preset-es2015/).
```

Comment:
I used to put absolute links to the website, otherwise the link on NPM will be broken.